website/docs/r/datadog_monitors.html.markdown

```diff
@@ -0,0 +1,146 @@
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this datadog Monitor. Changing this forces a new datadog Monitor to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the datadog Monitor should exist. Changing this forces a new datadog Monitor to be created.
+
+* `location` - (Required) The Azure Region where the datadog Monitor should exist. Changing this forces a new datadog Monitor to be created.
+
+* `identity` - (Required) A `identity` block as defined below.
+
+* `sku` - (Required) A `sku` block as defined below.
+
+* `user_info` - (Required) A `user_info` block as defined below.
+
+---
+
+* `datadog_organization_properties` - (Required) A `datadog_organization_properties` block as defined below.
+
+* `monitoring_status` - (Optional) Flag specifying if the resource monitoring is enabled or disabled. Possible values are "true" and "false" is allowed.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the datadog Monitor.
+
+---
+
+A `datadog_organization_properties` block exports the following:
+
+* `api_key` - (Required) Api key associated to the Datadog organization. Changing this forces a new datadog Monitor to be created.
+
+* `application_key` - (Required) Application key associated to the Datadog organization. Changing this forces a new datadog Monitor to be created.
+
+* `enterprise_app_id` - (Optional) The ID of the enterprise_app.
+
+* `linking_auth_code` - (Optional) The auth code used to linking to an existing datadog organization.
+
+* `linking_client_id` - (Optional) The ID of the linking_client.
+
+* `redirect_uri` - (Optional) The redirect uri for linking.
+
+---
+
+An `identity` block supports the following:
+
+* `type` - (Required) Specifies the identity type of the datadog Monitor. At this time the only allowed value is `SystemAssigned`.
+
+> **NOTE:** The assigned `principal_id` and `tenant_id` can be retrieved after the identity `type` has been set to `SystemAssigned` and the datadog Monitor has been created. More details are available below.
+---
+
+An `sku` block exports the following:
+
+* `name` - (Required) The name which should be used for this sku.
+
+---
+
+An `user_info` block exports the following:
+
+* `name` - (Required) The name which should be used for this user_info.
+
+* `email_address` - (Required) Email of the user used by Datadog for contacting them if needed. Changing this forces a new datadog Monitor to be created.
+
+* `phone_number` - (Optional) Phone number of the user used by Datadog for contacting them if needed.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the datadog Monitor.
+
+* `identity` - A `identity` block as defined below.
+
+* `liftr_resource_category` - The catogory of resource logs flowing.
+
+* `liftr_resource_preference` - The priority of the resource.
```

Comment:
if this is priority shouldn't it be
```suggestion
* `liftr_resource_priority` - The priority of the resource.
```

also what is liftr?

Reply:
Removed the property as it is not required.
Liftr is our internal team, present in swagger as well, will update the name.